app/src/main/java/com/theuhooi/uhooipicbook/extensions/ImageViewExtension.kt

```diff
@@ -0,0 +1,10 @@
+package com.theuhooi.uhooipicbook.extensions
+
+import android.widget.ImageView
+import androidx.databinding.BindingAdapter
+import coil.api.load
+
+@BindingAdapter("load")
```

Comment:
公式ドキュメントを読んだ感じだと、 `@BindingAdapter` アノテーションの `value` の名前は、関数名でなく属性名がよさそうです！（違ったらすみません）

```suggestion
@BindingAdapter("imageUrl")
```

Reply:
確かにその方がわかりやすいですね！